### ISSUE TITLE
PHP frontend incompatible with PHP 7

### ISSUE THREAD

**Issue by dopeghoti** [jamesoff/eggdrop-scripts#4]
I've fixed this on my deployment, I'll submit a PR if one would be welcome.

**Comment by jamesoff** [comment]
Certainly; the php for the QuoteEngine is probably the worst PHP I've ever written so fixes to it are always welcome!

**Comment by brandon15811** [comment]
I also made a rewritten version a few years ago that'll probably work with PHP 7 https://github.com/brandon15811/QuoteEngineWeb

**Comment by jamesoff** [comment]
Thanks! I've added a link to your version in the README, hope that's ok.

**Comment by brandon15811** [comment]
Yep, it's okay!

**Comment by Lord255** [comment]
i tried to use that 3rd party webpage, but couldn't make it work. :|
i have up around twig.. i followed the instructions, but couldn't make it work :|
